tests/test_package_addons.py

```diff
@@ -80,6 +80,33 @@ def test_mender_configure(
             "test -x /usr/share/mender/inventory/mender-inventory-mender-configure"
         )
 
+    def test_mender_container_modules(
+        self, setup_tester_ssh_connection, mender_dist_packages_versions
+    ):
+        # Upload
+        upload_deb_package(
+            setup_tester_ssh_connection,
+            mender_dist_packages_versions["mender-container-modules"],
+            "mender-docker-compose",
+            "all",
+        )
+
+        # Install
+        setup_tester_ssh_connection.run(
+            "sudo apt -y install --fix-broken ./"
+            + package_filename(
+                mender_dist_packages_versions["mender-container-modules"],
+                "mender-docker-compose",
+                "all",
+            )
+        )
+        check_installed(setup_tester_ssh_connection, "mender-docker-compose")
+
+        # Check mender-configure files
+        setup_tester_ssh_connection.run(
+            "test -x /usr/share/mender/modules/v3/docker-compose"
+        )
```

Comment:
This made me think about something! We are missing to install (and then check in the test) the directory in `/data/mender-docker-compose`.